betterrolls5e/scripts/chat-message.js

```diff
@@ -354,7 +354,10 @@ export class BetterRollsChatCard {
 			}
 
 			const roll = this.roll;
-			const item = await roll.getItem();
+			let item = await roll.getItem();
+			if (data.ammo) {
+				item = item.parent.items.get(item.data.data.consume.target)
```

Comment:
What I meant by the comment was that you should probably check that the apply active effects button is for the ammo before doing this replacement. I get the feeling that the button also does AE's for the ammo and never for the weapon if ammo is set.

Unless of course that's what data is.

Reply:
That is what it is, got it.